elements/media-quote/src/media-quote.js

```diff
@@ -23,6 +24,241 @@ class MediaQuote extends LitElement {
+            display: flex;
+            align-items: center;
+            padding: var(--ddd-spacing-0) var(--ddd-spacing-2);
+          }
+
+          .controller-text {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+          }
+
+          .controller:hover .show-hide, .controller:focus .show-hide {
+            text-decoration: underline;
+          }
+
+          .caption-content {
+            display: none;
+          }
+
+          figcaption {
+            color: var(--ddd-theme-default-potentialMidnight); 
+            width: 98%;
+            padding: var(--ddd-spacing-2);
+          }
+
+          @container (max-width: 1261px) and (min-width: 1000px) {
+            .quote {
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+          }
+
+          @container (max-width: 999px) { /* Mobile devices */
+            .text-overlay {
+              position: relative;
+              width: 90%;
+              top: 0%;
+              left: 0%;
+              font-size: var(--ddd-font-size-xs);
+            }
+
+            .text-overlay {
+              text-align: center;
+            }
+
+            .author, .author-detail {
+              font-size: var(--ddd-font-size-xxs);
+            }
+
+            figure {
+              width: 100%;
+            }
+
+            .content {
+              box-decoration-break: unset;
+              display: block;
+            }
+
+            .content, .citation {
+              padding: var(--ddd-spacing-2) var(--ddd-spacing-2);
+            }
+
+            figcaption {
+              width: 95%;
+            }
+          }
+      `,
+    ];
+  }
+
+  render() {
+    return html`
+        <div class="media-quote-container">
+          <figure>
+            <div class="top-content">
+              <div class="text-overlay"> 
+                <p class="quote">
+                  <span class="content"><slot name="quote"></slot></span>
+                    ${this.querySelector('[slot="author"]') && this.querySelector('[slot="author"]').textContent.trim().length > 0 ? html`
+                      <span class="citation">
+                        <span class="author">- <slot name="author"></slot></span> 
+                        ${this.querySelector('[slot="author-detail"]') && this.querySelector('[slot="author-detail"]').textContent.trim().length > 0 ? html`
+                          <span class="author-detail">, <slot name="author-detail"></slot></span>
+                        ` : ''}
+                      </span>
+                    ` : ''}
+                </p>  
+              </div>
+              <img src="${this.src}" alt="${this.alt}">
+            </div>
+            ${this.querySelector('[slot="caption"]') && this.querySelector('[slot="caption"]').textContent.trim().length > 0 ? html`
```

Comment:
make a variable like `hasCaption` so that's leveraged in the template instead of a query selector inside the template. Just odd looking